src/govuk/components/error-summary/error-summary.yaml

```
@@ -10,11 +10,11 @@ params:
 - name: descriptionText
   type: string
   required: false
-  description: If `descriptionHtml` is set, this is not required. Text to use for the description of the errors. If `descriptionHtml` is provided,  `descriptionText` will be ignored.
+  description: Text to use for the description of the errors. If `descriptionHtml` is provided,  `descriptionText` will be ignored.
```

```suggestion
  description: Text to use for the description of the errors. If you set `descriptionHtml` instead, `descriptionText` will be ignored.
```

Slight tweak to avoid passive voice (and fix the double space).